client/cypress/support/app.po.ts

```diff
@@ -3,19 +3,7 @@ export class AppPage {
     return cy.visit('/');
   }
 
-  getAppTitle() {
-    return cy.get('.app-title');
-  }
-
-  getSidenavButton() {
-    return cy.get('.sidenav-button');
-  }
-
-  getSidenav() {
-    return cy.get('.sidenav');
-  }
-
-  getNavLink(navOption: 'Pantry' | 'Products') {
-    return cy.contains('[routerlink] > .mat-list-item-content', `${navOption}`);
+  getNavLink(navOption: 'Pantry' | 'Products' | 'Shopping List') {
+    return cy.contains('[routerlink] > .mat-list-item-content', navOption);
```

Comment:
Add `'Shopping List'` to the set of `navOption`s so that we can test that part of the toolbar as well.

I also replaced `'${navOption}'` with the simpler (and in this case equivalent) `navOption`.

Reply:
Thank you for all the changes, I really appreciate you.